packages/card/style.module.css

```diff
@@ -73,5 +73,5 @@
   align-items: center;
   justify-content: flex-end;
   color: var(--wpl-neutral-200);
-  padding: 20px;
+  padding: 24px;
```

Comment:
We may want padding-top: 12px here from what I am seeing in the design.

Reply:
Hmm...perhaps even `8px`? Noticing that is the gap between the content and the outer border of the arrow icon in the design.

Pushed a commit so you can compare. I also removed bottom padding on `<Content />` to reduce the spacing further to match.

Reply:
well can content include 24px padding on each side and cta has now padding top?

Reply:
Hmm....`<Content />` can't have `padding-bottom`, from what I can see, if CTA gets `8px` of `padding-top`.  Otherwise, the space between `<Content />` and the CTA grows too large vs the design 🤔 

Unless `<Content />` gets `4px` of `padding-bottom`, then the CTA gets `4px` of `padding-top`? But in that case the spacing between CTA and other possible children then becomes too small?🤔